appveyor.yml

```diff
@@ -1,5 +1,5 @@
 environment:
-  nodejs_version: "0.10"
+  nodejs_version: "6"
```

Comment:
When we upgraded atom/atom we didn't go through all the package projects to upgrade. Feel free to bump them on appveyor as req.